Makefile.nt

```diff
@@ -373,7 +373,8 @@ otherlibs/dynlink/dynlink.cmxa: otherlibs/dynlink/natdynlink.ml
 
 utils/config.ml: utils/config.mlp config/Makefile
 	@rm -f utils/config.ml
-	sed -e "s|%%LIBDIR%%|$(LIBDIR)|" \
+	# Convert forward-slashes in LIBDIR to backslashes
+	sed -e "s|%%LIBDIR%%|$(shell echo $(LIBDIR)|sed -e 's:/:\\\\\\\\\\\\\\\\:g')|" \
```

Comment:
You could simplify this to:
```
sed -e "s|%%LIBDIR%%|$(subst /,\\\\,$(LIBDIR))|" \
```
and avoid problems with spaces at the beginning and end of `$(LIBDIR)`.